src/host/tcp.h

```diff
@@ -190,6 +190,8 @@ namespace asynchost
         return false;
       }
 
+      uv_tcp_keepalive(&uv_handle, 1, 30);
```

Comment:
We should check the return code of this (see http://docs.libuv.org/en/v1.x/guide/basics.html#error-handling).

Reply:
thanks!